kythe/cxx/indexer/proto/testdata/proto_verifier_test.bzl

```diff
@@ -14,31 +14,53 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+load("@bazel_skylib//lib:collections.bzl", "collections")
 load(
     "@io_kythe//tools/build_rules/verifier_test:verifier_test.bzl",
     "KytheVerifierSources",
     "extract",
     "index_compilation",
     "verifier_test",
 )
+load("@rules_proto//proto:defs.bzl", "ProtoInfo")
 
 def _invoke(rulefn, name, **kwargs):
     """Invoke rulefn with name and kwargs, returning the label of the rule."""
     rulefn(name = name, **kwargs)
     return "//{}:{}".format(native.package_name(), name)
 
+def get_proto_files_and_proto_path_opts(protolibs):
+    """Given a list of proto_library targets, returns a list of all transitively
+    -included .proto files and a list of options to pass to protoc (containing
+    --proto_path locations).
+    """
+    srcs = []
+    proto_paths = []
+    for lib in protolibs:
+        if ProtoInfo not in lib:
```

Comment:
just add a `providers` requirement on the attribute: https://docs.bazel.build/versions/master/skylark/lib/attr.html#label